invoicing_app/tests/tests.py

```diff
@@ -17,7 +16,8 @@
 from factories.order import OrderFactory
 
 
-class TestScriptGenerateTaxHandle(TestCase):
+# Unittest for the old script
```

Comment:
nit: In Python we use `docstrings` for this kind of comments. It would be something like:
```python
class TestScriptGenerateTaxReceiptsOld(TestCase):
"""Unittest for the old script"""
```

This way, most IDEs will show that comment when you are accessing this class